.github/workflows/build_package.yml

```diff
@@ -251,3 +251,16 @@ jobs:
         with:
           release_id: ${{ github.event.inputs.release_id }}
           assets_path: ./bindist/*
+
+  validate_and_publish:
+    name: "Invoke workflow to validate and publish release"
+    needs: build_core
```

Comment:
How about we also flip the `experimental: true` bit to false for those builds that are currently passing?

For example, 
* https://github.com/google/iree/runs/3271008495?check_suite_focus=true failed on `macos-latest :: Build main-dist Package`
* https://github.com/google/iree/runs/3299480891?check_suite_focus=true failed on `windows-2019 :: Build main-dist Package`

and both of those normally succeed (so we should mark them as non-experimental and triage/fix issues, or deflake the builds as needed)